assets/root-files/Authentication/phpunit.xml.dist

```diff
@@ -0,0 +1,45 @@
+<phpunit bootstrap="./test/bootstrap.php" colors="true">
```

Comment:
Will be nice if apply the changeset https://github.com/Maks3w/zf2/commit/04fd64b7e6f563939e99f9b3a7f22252fc589926

This will help to validate xml file.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="../vendor/phpunit/phpunit/phpunit.xsd"
         bootstrap="test/bootstrap.php"
         colors="true">
</phpunit>
```